Error while trying to add new row in empty table

In addition, it will be nice if there was some message "Table Empty" instead of error

Full panic error:
```
panic: runtime error: index out of range [0] with length 0 [recovered]
        panic: runtime error: index out of range [0] with length 0
goroutine 1 [running]:
github.com/rivo/tview.(*Application).Run.func1()
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/application.go:251 +0x4d
panic({0x782400, 0xc0006a6c78})
        /usr/local/go/src/runtime/panic.go:884 +0x213
github.com/jorgerojas26/lazysql/components.(*ResultsTable).tableInputCapture(0xc000266d80, 0xc0006ddc60)
        /Users/jorgerojas/projects/lazysql/components/ResultsTable.go:448 +0x1468
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc000270720?, 0xc0005c7b08?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:164 +0x3a
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc0005c7b48?, 0x54e0bf?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/flex.go:255 +0xd9
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0002701e0?, 0xc0005c7b88?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Pages).InputHandler.func1(0xc0005c7bc8?, 0x558dbf?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/pages.go:311 +0xa7
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc000270240?, 0xc0005c7c08?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Pages).InputHandler.func1(0xc0005c7c48?, 0x558dbf?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/pages.go:311 +0xa7
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0000fe5d0?, 0xc0005c7c88?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc0005c7cc8?, 0x54e0bf?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/flex.go:255 +0xd9
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc0000fe630?, 0x76d6e0?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Flex).InputHandler.func1(0xc0006ef7e0?, 0xc0005c7d48?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/flex.go:255 +0xd9
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc00007f2f0?, 0x7620e0?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Pages).InputHandler.func1(0x40d507?, 0x10?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/pages.go:311 +0xa7
github.com/rivo/tview.(*Box).WrapInputHandler.func1(0xc00007f110?, 0xc0005c7e40?)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/box.go:167 +0x53
github.com/rivo/tview.(*Application).Run(0xc00017e000)
        /Users/jorgerojas/.go/pkg/mod/github.com/rivo/tview@v0.0.0-20240101144852-b3bd1aa5e9f2/application.go:344 +0x57d
main.main()
        /Users/jorgerojas/projects/lazysql/main.go:16 +0x10a
```

--- a/components/ResultsTable.go
+++ b/components/ResultsTable.go
@@ -256,11 +256,59 @@ func (table *ResultsTable) tableInputCapture(event *tcell.EventKey) *tcell.Event
 			table.UpdateRows(table.GetIndexes())
 		}
 	}
+
+	command := app.Keymaps.Group("table").Resolve(event)
+
+	if command == commands.AppendNewRow {
+		if table.Menu.GetSelectedOption() == 1 {
+
+			newRow := make([]string, table.GetColumnCount())
+			newRowIndex := table.GetRowCount()
+			newRowUuid := uuid.New()
+
+			for i := 0; i < table.GetColumnCount(); i++ {
+				newRow[i] = "Default"
+			}
+
+			table.InsertRow(newRow, newRowIndex, newRowUuid)
+
+			for i := 0; i < table.GetColumnCount(); i++ {
+				table.GetCell(newRowIndex, i).SetBackgroundColor(tcell.ColorDarkGreen)
+			}
+
+			newInsert := models.DbInsert{
+				Table:           table.GetDBReference(),
+				Columns:         table.GetRecords()[0],
+				Values:          newRow,
+				PrimaryKeyValue: newRowUuid,
+				Option:          1,
+			}
+
+			*table.state.listOfDbInserts = append(*table.state.listOfDbInserts, newInsert)
+
+			if table.Tree.GetCurrentNode().GetColor() == tview.Styles.InverseTextColor || table.Tree.GetCurrentNode().GetColor() == tview.Styles.PrimaryTextColor {
+				table.Tree.GetCurrentNode().SetColor(InsertColor)
+			} else if table.Tree.GetCurrentNode().GetColor() == DeleteColor {
+				table.Tree.GetCurrentNode().SetColor(ChangeColor)
+			}
+
+			table.Select(newRowIndex, 0)
+
+			App.ForceDraw()
+			table.StartEditingCell(newRowIndex, 0, func(newValue string, row, col int) {
+				cellReference := table.GetCell(row, 0).GetReference()
+
+				if cellReference != nil {
+					table.MutateInsertedRowCell(cellReference.(uuid.UUID), col, newValue)
+				}
+			})
+
+		}
+	}
+
 	if rowCount == 1 || colCount == 0 {
 		return nil
 	}
-
-	command := app.Keymaps.Group("table").Resolve(event)
 
 	if command == commands.Search {
 		if table.Editor != nil {
@@ -431,51 +479,6 @@ func (table *ResultsTable) tableInputCapture(event *tcell.EventKey) *tcell.Event
 			} else {
 				table.AppendNewChange("DELETE", table.GetDBReference(), selectedRowIndex, -1, "")
 			}
-
-		}
-	} else if command == commands.AppendNewRow {
-		if table.Menu.GetSelectedOption() == 1 {
-
-			newRow := make([]string, table.GetColumnCount())
-			newRowIndex := table.GetRowCount()
-			newRowUuid := uuid.New()
-
-			for i := 0; i < table.GetColumnCount(); i++ {
-				newRow[i] = "Default"
-			}
-
-			table.InsertRow(newRow, newRowIndex, newRowUuid)
-
-			for i := 0; i < table.GetColumnCount(); i++ {
-				table.GetCell(newRowIndex, i).SetBackgroundColor(tcell.ColorDarkGreen)
-			}
-
-			newInsert := models.DbInsert{
-				Table:           table.GetDBReference(),
-				Columns:         table.GetRecords()[0],
-				Values:          newRow,
-				PrimaryKeyValue: newRowUuid,
-				Option:          1,
-			}
-
-			*table.state.listOfDbInserts = append(*table.state.listOfDbInserts, newInsert)
-
-			if table.Tree.GetCurrentNode().GetColor() == tview.Styles.InverseTextColor || table.Tree.GetCurrentNode().GetColor() == tview.Styles.PrimaryTextColor {
-				table.Tree.GetCurrentNode().SetColor(InsertColor)
-			} else if table.Tree.GetCurrentNode().GetColor() == DeleteColor {
-				table.Tree.GetCurrentNode().SetColor(ChangeColor)
-			}
-
-			table.Select(newRowIndex, 1)
-
-			App.ForceDraw()
-			table.StartEditingCell(newRowIndex, 1, func(newValue string, row, col int) {
-				cellReference := table.GetCell(row, 0).GetReference()
-
-				if cellReference != nil {
-					table.MutateInsertedRowCell(cellReference.(uuid.UUID), col, newValue)
-				}
-			})
 
 		}
 	}

